### PR TITLE
help text + ui improvement

### PR DESCRIPTION
**Pull request by InsafKhamzin**
added help texts. Keeping them in a separate service.

### CODE REVIEW

File: src/services/textService.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default class TextService {
+  _data = {
+    montly_deposit_help:
+      'This is the amount you are ready to invest monthly. A consistent monthly contribution is the most profitable and safest way to invest in the crypto market',
+    risklevel_help:
+      "The Risk Level is simply volatility you are ready to have for your portfolio. The more your portfolio volatile, the higher gains you're likely to have. But keep in mind, if the market fall, you're likely to experience larger losses.",
+    contibutions_help:
+      'Based on your risk level, these are contributions you should make monthly',
+    expectation_help:
+      "This expectation chart is based on your monthly contribution and your portfolio's risk level. It assumes that you will be investing monthly in the next 5 years",
```
Comment (dsych) [inline src/services/textService.js]: should say `for the next 5 years`

File: src/services/textService.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export default class TextService {
+  _data = {
+    montly_deposit_help:
+      'This is the amount you are ready to invest monthly. A consistent monthly contribution is the most profitable and safest way to invest in the crypto market',
+    risklevel_help:
+      "The Risk Level is simply volatility you are ready to have for your portfolio. The more your portfolio volatile, the higher gains you're likely to have. But keep in mind, if the market fall, you're likely to experience larger losses.",
```
Comment (dsych) [inline src/services/textService.js]: should be `falls`

File: src/services/textService.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export default class TextService {
+  _data = {
```
Comment (dsych) [inline src/services/textService.js:2]: what do you think about supporting languages other than english?

**Review by dsych — CHANGES_REQUESTED** [review]

File: src/services/textService.js (hunk 1 above)
Comment (InsafKhamzin) [reply src/services/textService.js]: Fixed

File: src/services/textService.js (hunk 2 above)
Comment (InsafKhamzin) [reply src/services/textService.js]: Fixed

File: src/services/textService.js (hunk 3 above)
Comment (InsafKhamzin) [reply src/services/textService.js:2]: yep that's why I put text into separate TextService. We could modify it to return different text depending on user's language later on.

**Comment by dsych** [comment]
@InsafKhamzin please sync the repo and we can merge

**Pull request merged**